explorer/app/viewModelBuilders/azul/hca-dcp/common/dataSummaryMapper/dataSummaryMapper.ts

```diff
@@ -149,42 +151,107 @@ export function mapProjectDataSummary(
+      label: getPluralizedMetadataLabel(METADATA_KEY.SAMPLE_TYPE),
+      values: sampleEntityType,
+    })
   ); // Sample Type
-  details.set(DATA_SUMMARY.ORGAN, stringifyValues(organ)); // Anatomical Entity
-  details.set(DATA_SUMMARY.ORGAN_PART, stringifyValues(organPart)); // Organ Part
+  details.set(
+    DATA_SUMMARY.ORGAN,
+    NTagCell({
+      label: getPluralizedMetadataLabel(METADATA_KEY.ANATOMICAL_ENTITY),
+      values: organ,
+    })
+  ); // Anatomical Entity
+  details.set(
+    DATA_SUMMARY.ORGAN_PART,
+    NTagCell({
+      label: getPluralizedMetadataLabel(METADATA_KEY.ORGAN_PART),
+      values: organPart,
+    })
+  ); // Organ Part
   details.set(
     DATA_SUMMARY.SELECTED_CELL_TYPE,
-    stringifyValues(selectedCellType)
+    NTagCell({
+      label: getPluralizedMetadataLabel(METADATA_KEY.SELECTED_CELL_TYPE),
+      values: selectedCellType,
+    })
   ); // Selected Cell Types
   // Model organ should only display a value when sampleEntityType is cellLines or organoids i.e. "modelOrgan" will
   // not display if the sampleEntityType is "specimens".
   if (!isSampleEntityTypeSpecimens(sampleEntityType)) {
-    details.set(DATA_SUMMARY.MODEL_ORGAN, stringifyValues(modelOrgan)); // Model Organ
+    details.set(
+      DATA_SUMMARY.MODEL_ORGAN,
+      NTagCell({
+        label: getPluralizedMetadataLabel(METADATA_KEY.MODEL_ORGAN),
+        values: modelOrgan,
+      })
+    ); // Model Organ
   }
-  details.set(DATA_SUMMARY.DISEASE, stringifyValues(disease)); // Disease Status (Specimen)
-  details.set(DATA_SUMMARY.DONOR_DISEASE, stringifyValues(donorDisease)); // Disease Status (Donor)
+  details.set(
+    DATA_SUMMARY.DISEASE,
+    NTagCell({
+      label: getPluralizedMetadataLabel(METADATA_KEY.DISEASE_STATUS_SPECIMEN),
+      values: disease,
+    })
+  ); // Disease Status (Specimen)
+  details.set(
+    DATA_SUMMARY.DONOR_DISEASE,
+    NTagCell({
+      label: getPluralizedMetadataLabel(METADATA_KEY.DISEASE_STATUS_DONOR),
+      values: donorDisease,
+    })
+  ); // Disease Status (Donor)
   details.set(
     DATA_SUMMARY.DEVELOPMENT_STAGE,
-    stringifyValues(developmentStage)
+    NTagCell({
+      label: getPluralizedMetadataLabel(METADATA_KEY.DEVELOPMENT_STAGE),
+      values: developmentStage,
+    })
   ); // Development Stage
   details.set(
     DATA_SUMMARY.LIBRARY_CONSTRUCTION_APPROACH,
-    stringifyValues(libraryConstructionApproach)
+    NTagCell({
+      label: getPluralizedMetadataLabel(
+        METADATA_KEY.LIBRARY_CONSTRUCTION_APPROACH
+      ),
+      values: libraryConstructionApproach,
+    })
   ); // Library Construction Method
   details.set(
     DATA_SUMMARY.NUCLEIC_ACID_SOURCE,
-    stringifyValues(nucleicAcidSource)
+    NTagCell({
+      label: getPluralizedMetadataLabel(METADATA_KEY.NUCLEIC_ACID_SOURCE),
+      values: nucleicAcidSource,
+    })
   ); // Nucleic Acid Source
-  details.set(DATA_SUMMARY.PAIRED_END, stringifyValues(pairedEnd)); // Paired End
+  details.set(
```

Comment:
Because Paired End is only a set of three possible values, let's revert and just stringify the values.